pihole/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "pihole",
+  "maintainer": "monganai@tcd.ie",
+  "manifest_version": "1.0.0",
+  "name": "pihole",
+  "metric_prefix": "pihole.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "In tegration to collect default pihole metrics",
+  "guid": "f0a69a1e-3961-43e2-9848-469342734e34",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-pihole Integration",
+  "categories": [
+    ""
```

Comment:
no categories ? you can found the available one here: https://docs.datadoghq.com/integrations/

Reply:
👍